epm/pkg/epm/enclave-cache-pool/enclavepool/enclave-cache.go

```diff
@@ -13,6 +13,10 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	EPMDir string = "/var/run/containerd"
```

Comment:
Why use `/var/run/containerd` dir? `/var/run/containerd` is a default dir of containerd but a constant. Users can change it   in the `containerd` configuration file(such as `/etc/containerd/config.toml`) .